docs/Permissions/Onchain-Permissioning/Getting-Started-Onchain-Permissioning.md

```diff
@@ -101,6 +105,9 @@ to enable onchain nodes permissioning
 * [--permissions-nodes-contract-address](../../Reference/Pantheon-CLI-Syntax.md#permissions-nodes-contract-address)
 set to the address of the Node Ingress contract in the genesis file (`"0x0000000000000000000000000000000000009999"`)  
 
+Start your first node with command line options to enable onchain permissioning and the JSON-RPC HTTP host and port 
```

Comment:
Start your first node with command line options to enable onchain permissioning, the JSON-RPC HTTP host, and...

Reply:
The JSON-RPC host and port must match the environment variable so leaving the and where it is.